Monika After Story/game/zz_calendar.rpy

```diff
@@ -626,7 +646,7 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
                         self.DAY_NUMBER_DISPLAY_FORMAT.format(str(current_date.day)),
                         font=gui.default_font,
                         size=self.DAY_NUMBER_TEXT_SIZE,
-                        color=self.DAY_NUMBER_COLOR,
+                        color=self.DAY_NUMBER_COLOR if not is_today else self.TODAY_NUMBER_COLOR,
```

Comment:
this is now 4 additional branches per button creation. Can you move these so you just check `is_today` once per button, set temp vars as appropriate, and pass them into the `MASButtonDisplayable` accordingly?

Reply:
Done.

(The amount of `_bg`s is becoming a little bit messy, though.)